frontend/src/pages/modelServing/screens/global/InternalServicePopoverContent.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import {
+  DescriptionList,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  Stack,
+  StackItem,
+} from '@patternfly/react-core';
+import { InferenceServiceKind } from 'k8sTypes';
+
+type InternalServicePopoverContentProps = {
+  inferenceService: InferenceServiceKind;
+};
+
+const InternalServicePopoverContent: React.FC<InternalServicePopoverContentProps> = ({
+  inferenceService,
+}) => {
+  const isInternalServiceEnabled = inferenceService.status?.components?.predictor;
+
+  if (!isInternalServiceEnabled) {
+    return (
+      <Stack hasGutter>
+        <StackItem>Could not find any internal service enabled</StackItem>
+      </Stack>
```

Comment:
Huh? What are you guttering?

No harm -- just odd.

Reply:
mmmm you are right, i refactored this, first I intended to have more info in the error and then discarded it talking with Vince, might delete it in a follow up.